tilequeue/queue/redis_queue.py

```diff
@@ -41,6 +41,9 @@ def read(self):
             msg_handles.append(msg_handle)
         return msg_handles
 
+    def job_paritally_done(self, handle):
```

Comment:
Spelling mistake in `paritally`.

Reply:
Doh! Good catch. Updated in facde1f also.